libdnf/conf/option_string.cpp

```diff
@@ -64,7 +66,7 @@ void OptionString::test(const std::string & value) const {
         regex,
         std::regex::nosubs | std::regex::extended | (icase ? std::regex::icase : std::regex_constants::ECMAScript));
     if (!std::regex_match(value, re)) {
-        throw NotAllowedValue(value);
+        throw OptionValueNotAllowedError(M_("Input value \"{}\" does not match the regular expression \"{}\""), value, regex);
```

Comment:
Seems I overlooked this one (should be the same as `OptionStringList`):
> "Input value \\"{}\\" not allowed, allowed values for this option are defined by regular expression \\"{}\\""

Reply:
Ok.